### ISSUE TITLE
Requires_grad is False

### ISSUE THREAD

**Issue by BornaGhotbi** [facebookresearch/inversecooking#3]
Hi,

I guess the code needs to have some change in line 390 from src/modules/transformer_decoder.py
logits = torch.stack(logits, 1).data --> logits = torch.stack(logits, 1)

This will enable the require_grad to remain true. The training cannot do the backward.loss process without the require_grad being True.

**Comment by amaiasalvador** [comment]
Fixed by 3ab629c08f0b5d84bbc3d56ac70ce12e60500322 .
